lib/vty.c

```diff
@@ -345,8 +345,16 @@ int vty_out(struct vty *vty, const char *format, ...)
 	case VTY_SHELL_SERV:
 	case VTY_FILE:
 	default:
+		vty->vty_buf_size_written += strlen(filtered);
 		/* print without crlf replacement */
 		buffer_put(vty->obuf, (uint8_t *)filtered, strlen(filtered));
+		/* For every 32 *4 KB invoke vtysh_flush where we
```

Comment:
I'll repeat the earlier comment: please don't hard-code values like these into comments. over time, they may become stale, and confusing.